src/AspNetCoreModuleV2/OutOfProcessRequestHandler/forwardinghandler.cpp

```diff
@@ -5,6 +5,7 @@
 #include "url_utility.h"
 #include "exceptions.h"
 #include "ServerErrorHandler.h"
+#include "../AspNetCore/resource.h"
```

Comment:
You shouldn't need to include anything in AspNetCore into the oop request handler.